data/strings/strings.txt

```diff
@@ -20860,48 +20860,48 @@
 
   [placepage_remove_stop]
     tags = android,ios
-    en = Remove
-    af = Verwyder
-    ar = إزالة
-    az = Sil
-    be = Выдаліць
-    bg = Премахване
-    ca = Elimina
-    cs = Odstranit
-    da = Fjern
-    de = Entfernen
-    el = Αφαίρεση
-    es = Eliminar
-    et = Eemalda
-    eu = Kendu
-    fa = حذف
-    fi = Poista
-    fr = Supprimer
-    he = הסרה
-    hu = Eltávolítja
-    id = Hapus
-    it = Rimuovi
-    ja = 削除
-    ko = 제거
-    lt = Šalinti
-    lv = Noņemt
-    mr = काढा
-    nb = Fjern
-    nl = Verwijder
-    pl = Usuń
-    pt = Remover
-    pt-BR = Remover
-    ro = Elimină
-    ru = Удалить
-    sk = Odstrániť
-    sv = Ta bort
-    sw = Ondoa
-    th = เอาออก
-    tr = Kaldır
-    uk = Видалити
-    vi = Xóa
-    zh-Hans = 移除
-    zh-Hant = 移除
+    en = Remove Stop
+    af = Verwyder stop
+    ar = إزالة نقطة توقف
+    az = Kəsmə nöqtəsini silin
+    be = Выдаліць прыпынак
+    bg = Премахване на спирка
+    ca = Elimina parada
+    cs = Odstranit zastávku
+    da = Fjern stop
+    de = Stopp entfernen
+    el = Αφαίρεση στάσης
+    es = Eliminar parada
+    et = Eemalda peatus
+    eu = Kendu geldiunea
+    fa = توقف را حذف کنید
+    fi = Poista pysähdys
+    fr = Supprimer arrêt
+    he = הסר את העצירה
```

Comment:
```suggestion
    he = הסר עצירה
```

and `placepage_add_stop` should be changed to:
```
    he = הוסף עצירה
```

Reply:
Thanks @NeatNit: Modified both strings !!